src/pickWithDefault.js

```diff
@@ -0,0 +1,36 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * Similar to `pick` except that this one includes a `key: <default>` pair for
+ * properties that don't exist
+ *
+ * **Note:** A property set to undefined will be left untouched.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.22.2
+ * @category Object
+ * @sig [k] -> * -> {k: v} -> {k: v}
```

Comment:
> I tried keeping it exactly like pickAll

Yep. Our signatures need some attention. There's an argument for keeping things consistent until we update *all* the signatures, but that's a daunting prospect. I'm happy to make small improvements here and there.

> I don't see 'Array String' anywhere but i do see ['[String]'](http://ramdajs.com/0.22.1/docs/#mapObjIndexed).

I switched to using `Array a` rather than `[a]` in Sanctuary, but Ramda uses the latter. For Ramda I'd like:

```haskell
pickWithDefault :: [String] -> a -> StrMap a -> StrMap a
```

As far as I'm aware we're not yet using `StrMap` in any of our signatures. I'm interested to know whether @buzzdecafe and @CrossEye are interested in adopting it.